packages/ember-views/lib/views/collection_view.js

```diff
@@ -357,13 +357,35 @@ var CollectionView = ContainerView.extend({
         item = content.objectAt(idx);
 
         itemViewProps.content = item;
-        itemViewProps._blockArguments = [item];
+        if (!Ember.FEATURES.isEnabled('ember-each-with-index')) {
```

Comment:
Due to a limitation of `defeatureify` we cannot use `!Ember.FEATUREs.isEnabled`, you have to use:

```javascript
if (Ember.FEATURES.isEnabled('blah')) {

} else {
  /* your stuff here */
}
```